scripts/server.js

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env node
+'use strict';
+
+const execSync = require('child_process').execSync;
+
+let browserSync = require('browser-sync').create();
+
+execSync('npm run babel; npm run browserify', { stdio: [0, 1, 2] });
```

Comment:
`execSync` will, by default, return a stream of output that you process into data by watching for new chunks, processing them, and joining them together into a giant blob of output. This output is not viewable by the user until everything is finished. No thanks.

By telling `execSync` to ignore the streaming output in our program, and instead pass it to the `standard input/output` (`stdio`) of the parent caller (in this case, a command line shell such as "Terminal"), we can both simplify our program, as well as hand off a stream to be immediately processed by the command line interface that called it. i.e., no waiting for it to finish. It just prints the output as it goes, just like you'd expect.